src/stores/index.js

```diff
@@ -87,6 +88,8 @@ const useRootStore = defineStore('root', {
      */
     addCSVLogMessage(info, error, type = 'danger') {
       this.addMessage({ type, message: errorFor(info, error) }, { root: true })
```

Comment:
Is line 90 still needed?

Reply:
Yes, eventually we need to remove the Vuex store for messages (traction/addMessage - line 92) and use this store, so it's probably good to keep the Pinia store functioning.